src/App.js

```diff
@@ -71,87 +71,98 @@ const theme = {
       button: {
         link: {
           color: {
-            value: '#0d47a1'
-          }
+            value: '#0d47a1',
+          },
         },
         primary: {
           background: {
             color: {
-              value: '#0d47a1'
-            }
-          }
+              value: '#0d47a1',
+            },
+          },
         },
         _hover: {
           color: {
-            value: '#0d47a1'
-          }
-        }
+            value: '#0d47a1',
+          },
+        },
       },
       tabs: {
         item: {
           _hover: {
             color: {
-              value: '#0d47a1'
-            }
+              value: '#0d47a1',
+            },
           },
           _active: {
             color: {
-              value: '#0d47a1'
+              value: '#0d47a1',
             },
             border: {
               color: {
-                value: '#0d47a1'
-              }
+                value: '#0d47a1',
+              },
             },
           },
           _focus: {
             color: {
-              value: '#0d47a1'
+              value: '#0d47a1',
             },
-          }
-        }
-      }
-    }
-  }
-}
+          },
+        },
+      },
+    },
+  },
+};
 
 const MyApp = () => {
-  const config = useConfig()
-  setupDHIS2Api(config)
+  const config = useConfig();
+  setupDHIS2Api(config);
 
   const components = {
     Footer() {
       return (
         <Card>
-        <div className={classes.instructions}>
-          <p className={classes.instructionText}>{i18n.t("To use the Microplanning app, you need to log in to your Crosscut account. Don't have an account? You can create one for free by clicking the 'Create Account' tab above or visit ")}
-          <a style={{ color: '#0d47a1'}} href="https://app.crosscut.io/" target="_blank" >app.crosscut.io</a>.
-          </p>
+          <div className={classes.instructions}>
+            <p className={classes.instructionText}>
+              {i18n.t(
+                "To use the Microplanning app, you need to log in to your Crosscut account. Don't have an account? You can create one for free by clicking the 'Create Account' tab above or visit "
+              )}
+              <a
+                style={{ color: '#0d47a1' }}
+                href="https://app.crosscut.io/"
+                target="_blank"
+                rel="noreferrer"
+              >
+                app.crosscut.io
+              </a>
+              .
```

Comment:
This looks hard to read -- could the text be wrapped in the code without wrapping it in the display?